script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -1337,17 +1351,17 @@ I simply can't stand that her past,
 -- Page 125, Offset 29855.
 -- ……本当に、俺は半分しか答えを見つけていなかった。
 // Callback earlier in the file, offset 29646.
-...I guess I never had found the other half of that answer before.
+I really had only ever found half the answer...
 }
 [sha:990c0c0d84352c4f69619e04f86c208d9d9d27c9]{
 -- Page 125, Offset 29856.
 -- やるべき事なんて一生わからない。
-I've spent my whole life not knowing what to do.
+I will never in my whole life understand what I %{i}should%{/i} do.
```

Comment:
I don't agree with this interpretation, original seems correct to me.

Reply:
Nope!